src/javascript/app/ng-wig/ng-wig.component.spec.js

```diff
@@ -22,14 +22,16 @@ describe('component: ngWig', () => {
     };
     let mockWindow;
     let compile;
+    let $timeout;
```

Comment:
I pointed it out so that it conforms with the other variables above. Either using a $ or not, we should stick with a single naming convention.